### ISSUE TITLE
Add LBRY/Odysee as source

### ISSUE THREAD

**Issue by mlovizio** [nukeop/nuclear#1122]
As the title suggest I think it would be positive to add support for looking for content in the LBRY platform.

Is it possible?

**Comment by HaiDang666** [comment]
Yes, it is possible if it provides an endpoint to query and fetch stream URLs.

**Comment by nukeop** [comment]
I looked at npm and there are no published modules that would let us easily search and get audio streams. We can still write our own scraper or use their api (if there's a public one). Not sure if it's possible to get only audio, like it is with youtube.

**Comment by mlovizio** [comment]
As far as I know, LBRY supports all types of content, video but also audio, images, text. Maybe searching only for audio files would be a solid starting point, sorry for not being much help with the technical details :/

**Comment by nukeop** [comment]
This issue is from an older version of Nuclear that has been replaced with a complete rewrite. None of the old issues apply to the new codebase.

The old code is preserved on the [legacy/electron](https://github.com/nukeop/nuclear/tree/legacy/electron) branch. If you have feedback about the new version, head to [Discussions](https://github.com/nukeop/nuclear/discussions).
